ui/app/AppLayouts/Chat/panels/UserListPanel.qml

```diff
@@ -115,8 +125,8 @@ Item {
             delegate: StatusMemberListItem {
                 width: ListView.view.width
                 nickName: model.localNickname
-                userName: ProfileUtils.displayName("", model.ensName, model.displayName, model.alias)
-                pubKey: model.isEnsVerified ? "" : Utils.getCompressedPk(model.pubKey)
+                userName: model.preferredDisplayName
```

Comment:
This is wrong (unless you rewrite the logic inside `StatusMemberListItem`); notice the previous version omitted the `nickname` part), now you are including it. So the resulting delegate will have the nickname duplicated